contributing/ci-docs.txt

```diff
@@ -342,6 +357,37 @@ The repository for the OME help is https://github.com/openmicroscopy/ome-help.
 			#. rysnc the content of :file:`/ome/data_repo/public/help-staging`
 			   to :file:`/ome/data_repo/public/help`
 
+OMERO.figure website
+^^^^^^^^^^^^^^^^^^^^
+
+The repository for OMERO.figure is https://github.com/openmicroscopy/figure.
+
+.. glossary::
+
+	:jenkinsjob:`FIGURE-help-staging`
+
+		This job is used to review the PRs opened against the gh-pages-staging
+		branch of figure repository
+
+		#. |merge| (and also incorporates
+		   https://github.com/snoopycrimecop/figure/tree/cname_staging to
+		   allow  deployment to a non-GitHub URL) then pushes the resulting
+		   branch to https://github.com/snoopycrimecop/figure/tree/gh-pages
+		#. The GitHub Pages services updates the content of
+		   http://figure.staging.openmicroscopy.org
+
+		.. seealso:: :doc:`jekyll`
+
+	:jenkinsjob:`FIGURE-help-release`
+
+		This job is used to deploy the OME help documentation
+
+		#. Opens a Pull Request from
+		   https://github.com/openmicroscopy/figure/tree/gh-pages-staging
+		   to https://github.com/openmicroscopy/figure/tree/gh-pages. If
+		   this PR is merged, the GitHub Pages services updates the content of
```

Comment:
"he GitHub Pages services updates" -- should it be "services update" or "service updates"?